### DISCUSSION TITLE
Reflashing firmware ESP issue

### DISCUSSION THREAD

**Discussion by jmiguelsangar** [bitcraze/discussions#120] [Q&A] [answered]
Hi everyone,
After trying making the wifi example work without succeed (always hangs at 6% Writing to bcAI:gap8 deck memory), I have started from the begining reflashing all firmwares again. Following the guide https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/

STM and NRF reflashes without problem, but I cannot see reflashing ESP (‘bcAI:esp deck memory’) (The guide says it takes about 3 minutes)

The results are: STATUS is idle, ESP32: I (899) SYS: Initialized shows in the consoloe and LED1 flashes with 2 hz.

May I suppose the ESP has been flashed satisfactory even if the progress is not showed?

Thank you for the help in advance

**Comment by knmcguire** [comment]
Hi! Some additional questions about your problem:
- After the STM and NRF were reflashed, did the dialog stay on while it displayed bcAI:esp deck memory? Or did it turn to status 'Idle' right away after restarting.
- Did you had any other deck connected to the crazyflie then the AIdeck.

**Reply by jmiguelsangar** [reply]
Hi!
The dialog shows bcAI:esp deck... less than half a second after the reboot. It is very quick to dissapear. The progress bar does not show any progress.
Apart from the AIdeck, I have the flow deck v2 connected. I will try too reflash without it...

Thanks for your help!

**Comment by knmcguire** [comment, marked as answer]
@jmiguelsangar ah Yes that is it! It's because you have a second deck attached that it won't upload properly. We have an issue open for this: https://github.com/bitcraze/crazyflie-clients-python/issues/616 and the getting started tutorial has been updated.

Let us know if this solved your flashing problem.

**Reply by jmiguelsangar** [reply]
Hi, I had to make two trials but at the end it was reflashed. The problem was the flow deck attached. Now the wifi-stream example is working perfect. Just a reminder: I had to uninstall the cfclient to make opencv-viewer.py work. I have to reinstall it on a different python environment in case I need it later on as stated in https://github.com/bitcraze/crazyflie-clients-python/issues/611

Thank you very much for the support

**Reply by knmcguire** [reply]
Yes... that issue is indeed annoying. We put it into the tutorial as warning, but we hope to replace the opencv headless in cfclient soon. That one is used for the lighthouse geometery estimate.

Good that your initial problem is fixed! Could click 'mark as answer' in that case?
